scripts/pretty-printers/gdb/pretty_printers.py

```diff
@@ -42,8 +42,10 @@ def to_string(self):
             if value.is_optimized_out:
                 return "{}: <Optimized out>".format(string_no)
             return "{}: \"{}\"".format(string_no, value.string().replace("\0", "").replace("\"", "\\\""))
+        except RuntimeError as e:
+            return "Exception evaluating dstringt: {}".format(e.message)
         except:
-            return ""
+            return "Exception evaluating dstringt"
```

Comment:
I don't think showing the exception is useful to anyone unless they're debugging the pretty printers, better to just leave it blank so it doesn't show anything at all.

Reply:
You might be right. Though a blank string may be confusing to the user?

Reply:
When I tested it before it wasn't a blank string but no value to be printed, might've changed a little with the type also changing, but that'd be the aim.

Reply:
OK, dropped that commit